README.md

```diff
@@ -27,12 +27,19 @@
 
 # 1. What is Cloud Hypervisor?
 
-Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) that runs on top of [KVM](https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt) and the MSHV hypervisors .
+Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) that runs on
+top of [KVM](https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt) and
+the MSHV hypervisors.
```

Comment:
Reading `MSHV hypervisor` is like reading `ATM machine`. Since you're changing it anyway, can you change it to `Microsoft Hypervisor (MSHV)`. I know the `hypervisors` is plural here and refer to both KVM and MSHV, so the suggestion is more like dropping the plural form and expand MSHV to make the document flow better.

Reply:
Updated! Please check if it looks as you expected.